conf/logback.xml

```diff
@@ -4,7 +4,9 @@
   <conversionRule conversionWord="coloredLevel" converterClass="play.api.libs.logback.ColoredLevel" />
 
   <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-    <file>${application.home:-.}/logs/application.log</file>
+    <!-- There is a bug when using application.home with sbt dist - see https://github.com/playframework/playframework/issues/8759 -->
+    <!-- <file>${application.home:-.}/logs/application.log</file> -->
+    <file>/opt/ctd-omega-editorial-frontend/logs/application.log</file>
```

Comment:
But this will cause the follow error when that directory doesn't exist, like on a mac:

```
11:54:54,631 |-ERROR in ch.qos.logback.core.FileAppender[FILE] - openFile(/opt/ctd-omega-editorial-frontend/logs/application.log,true) call failed. java.io.FileNotFoundException: /opt/ctd-omega-editorial-frontend/logs/application.log (No such file or directory)
        at java.io.FileNotFoundException: /opt/ctd-omega-editorial-frontend/logs/application.log (No such file or directory)
```

Reply:
However, it doesn't prevent the app from running.

Reply:
One option might be to have a conf file for production usage, in the same directory (maybe `logback-production.xml`.

We would have to specify it, as explained [here](https://www.playframework.com/documentation/2.8.x/ProductionConfiguration#Logging-configuration).

Reply:
Good call.